src/N98/Util/Console/Helper/DatabaseHelper.php

```diff
@@ -32,6 +32,21 @@ class DatabaseHelper extends AbstractHelper
     protected $_tables;
 
     /**
+     * @var string
+     */
+    protected $_connectionType = 'default';
```

Comment:
Please do not signal visibility with the underscore character (even we have code that does it wrong in the project).